components/Card/Card.js

```diff
@@ -0,0 +1,39 @@
+import Image from "next/image";
+import styled from "styled-components";
+import dummyBook from "../../public/img/dummy/dummy_book.png";
+
+export default function Card({ coverimage, author, title, subtitle }) {
+  return (
+    <StyledCard>
+      {coverimage ? (
+        <Image src={coverimage} width={70} height={100} alt={title} priority />
+      ) : (
+        <Image
+          src={dummyBook}
+          width={70}
+          height={100}
+          alt="No picture"
+          priority
+        />
+      )}
+      <StyledBookOverview>
+        <h2>{author}</h2>
+        <h3>{title}</h3>
+        <h3>{subtitle}</h3>
```

Comment:
The subtitle should be an h4 here since it is of a lower hierarchy than the main title.

Reply:
Thanks, you are right. I will change this. But later i must change the h3 element with css, because in the view, it is to big.

Reply:
Sorry i mean to small.